src/trace.c

```diff
@@ -78,12 +78,12 @@ trace_error(const char *fmt, ...)
     vfprintf(stderr, fmt, args);
     va_end(args);
 
-    if (!stdout_is_a_tty) {
-        // If stdout is redirected to a file, make sure it also gets error messages.
+    if (!stderr_is_a_tty) {
+        // If stderr is redirected to a file, make sure it also gets error messages.
         // That helps to figure out where errors were.
-        fprintf(stdout, "[apulse] [error] ");
+        fprintf(stderr, "[apulse] [error] ");
```

Comment:
This duplicates error messages. There is already `fprintf()` that writes to stderr half a dozen lines earlier.